lib/fastly_nsq/message_queue.rb

```diff
@@ -8,4 +8,16 @@
 module MessageQueue
   FALSY_VALUES = [false, 0, '0', 'false', 'FALSE', 'off', 'OFF', nil]
   TRUTHY_VALUES = [true, 1, '1', 'true', 'TRUE', 'on', 'ON']
+
+  ##
+  # Passthru to wistia/nsq gem logger= method
+  def self.logger=(logger)
+    Strategy.for_queue.logger = logger
+  end
+
+  ##
+  # Passthru to wistia/nsq gem logger method
```

Comment:
This passes through to both the real and fake logger methods now, so consider removing all of these comments.

![code_comments](https://cloud.githubusercontent.com/assets/913757/13656332/cf757c38-e61a-11e5-9f67-dab5ddc49680.jpg)